api/core/v1alpha1/extension.proto

```diff
@@ -18,7 +18,7 @@ message Extension {
   // Name of the extension attached to an agent.
   string name = 1;
   // Version of the extension attached to an agent.
-  string version = 2;
+  optional string version = 2;
```

Comment:
perhaps we keep this as it was, but we can treat empty value as unset (ie. to come from schema version)